indexer/editable_map_object.cpp

```diff
@@ -744,6 +766,124 @@ bool EditableMapObject::ValidateWebsite(string const & site)
   return true;
 }
 
+//static
+bool EditableMapObject::ValidateFacebookPage(string const & facebookPage)
```

Comment:
nit: s/facebookPage/page

Verbose variables inside ValidateFacebookPage are not necessary.

Reply:
Renamed according to suggestion. I have Enterprise Java background and I usually give verbose names to methods and variables :)